packages/core/contracts/UnirepSocial.sol

```diff
@@ -28,10 +28,17 @@ contract UnirepSocial {
     // The amount of karma airdropped to user when user signs up and executes user state transition
     uint256 immutable public airdroppedReputation;
 
+    // The set of user names
+    uint256[] public userNames;
```

Comment:
We don't need an array of usernames. We shouldn't ever need to iterate over them